onadata/libs/utils/user_auth.py

```diff
@@ -1,18 +1,23 @@
 # coding: utf-8
 from __future__ import unicode_literals, print_function, division, absolute_import
 import base64
-from functools import wraps
 import re
+from functools import wraps
```

Comment:
Aren't these all part of the standard library?

Reply:
They are.

Reply:
ha, well i can't find the blank lines between standard library imports that i saw before. not sure what i was looking at :shrug: